### PR TITLE
Working on task

### PR DESCRIPTION
**Pull request by AsadThafer**
need to merge it

### CODE REVIEW

File: src/components/Checkout.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        totalprice +
+        "." +
+        " Your cart has been emptied." +
+        " You will receive an email at " +
+        email +
+        " shortly."
+    );
+    onCheckoutSuccessfull();
+    setFullname("");
+    setEmail("");
```
Comment (MohammadAbusaa) [inline src/components/Checkout.js]: this is already done in the callback above

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      return prevcart.filter((product) => product.id !== id);
+    });
+  };
+  const totalprice = cart.reduce(
+    (total, product) => total + product.price * product.count,
+    0
+  );
+
+  const HandleSuccessfullAdd = ({ id, title, price, image, rating, count }) => {
+    const product = {
```
Comment (MohammadAbusaa) [inline src/App.js]: you are destructing and then constructing an object again 🤔

**Review by MohammadAbusaa — COMMENTED** [review]
Make all changes please! thanks! goood luuuuuck!!!!

File: src/App.js (hunk 2 above)
Comment (AsadThafer) [reply src/App.js]: done it

**Review by MohammadAbusaa — APPROVED** [review]
Nice! thank you. I'm very proud of you! keep it up!

**Pull request merged**